common/predictive-text/index.ts

```diff
@@ -20,6 +20,11 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
+// Include the code intended to run WITHIN the Web Worker.
+// This MUST be compiled before this file is compiled.
+/// <reference path="embedded_worker.js" />
+declare var LMLayerWorkerCode: Function;
```

Comment:
I tried replicating `testing/two-stage-embedded-webworker`, and although `/// <reference path=...>` _does_ include the inner-Worker code, TypeScript refuses to acknowledge it. Not sure why.